ajax/libs/simple-slider/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "simple-slider",
+  "version": "1.0.0",
+  "description": "The 1kb JavaScript Carousel",
+  "npmName": "simple-slider",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "simpleslider.min.js",
+  "homepage": "http://ruyadorno.github.io/simple-slider/",
```

Comment:
@ruyadorno I think it's better to use `https` instead of `http` for github.io domain.